File: data/submissions/P001/sum_ac.js
// Bai nop dung cho P001 bang JavaScript (Node.js) -> ky vong AC.
const data = require("fs").readFileSync(0, "utf8").trim().split(/\s+/);
const a = BigInt(data[0]);
const b = BigInt(data[1]);
console.log((a + b).toString());

File: data/submissions/P001/sum_ac.rs
// Bai nop dung cho P001 bang Rust -> ky vong AC.
use std::io::Read;

fn main() {
    let mut input = String::new();
    std::io::stdin().read_to_string(&mut input).unwrap();
    let nums: Vec<i64> = input
        .split_whitespace()
        .map(|x| x.parse().unwrap())
        .collect();
    println!("{}", nums[0] + nums[1]);
}
